_posts/2022-11-30-trino-summit-2022-comcast-recap.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Elevating Data Fabric to Data Mesh: solving data needs in hybrid data lakes"
+author: "Sajuman Joseph, Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/comcast.jpg
+---
+
+Tune in, for the next [post in the Trino Summit 2022 recap series]({% post_url
+2022-11-21-trino-summit-2022-recap %}). In this post, we're joining "Saj" from
+Comcast, to talk about their migration from a data fabric to data mesh. Saj
+shows you there's more to the buzzword than meets the eye, and gives a solid
+overview of why Comcast is taking data mesh to heart.
+
+<!--more-->
+
+{% youtube sSWBi7bBotQ %}
+
+<a class="btn btn-pink btn-md" target="_blank" href="/assets/blog/trino-summit-2022/Trino@Comcast.pdf">
+  Check out the slides here!
+</a>
+
+## Recap
+
+Comcast engineer, Sajuman Joseph, brings us through the process they went
+through at Comcast to move from their initial use case of using Trino to power
+their data fabric architecture to include more governance features by
+leveraging Trino. Data fabric enables querying data across distributed data
+sets, but importantly, enabled Comcast to transparently migrate data across
+on-prem and cloud storage without any user knowledge.
+
+Despite offering query federation, data fabric still misses out on a higher
+quality experience that data mesh aims to solve. Not only having access to the
+data matters, but ensuring that the data has quality checks and a dedicated
+owner that ensures the data is correct and consumable. The ownership is split
+by domains defined by Comcast. It is the responsibility of the owners to ensure
+data quality, compliance, and security on the data they own. This data can be
+exposed internally or externally as a data product. While much of the drivers
+for this are done through company policy, there are technical means to make this
+possible. This starts from improving metadata on the data, access logs, global
+data catalogs, and managing data access.
+
+Trino facilitates the single point of access and is the one location where you
+will need to enforce a lot of these policies. Comcast created an engine called
+the Enterprise Policy Hub, that syncs with all data stores and compute engines
+to enforce company policy and update metadata on all data across Comcast. Trino
```

Comment:
We're going from future tense ("you will need to") to past ("Comcast created") very quickly here, which is a little jarring. I'd probably try to make the first sentence present tense.